### PR TITLE
Add an option to disable default keyboard mappings

### PR DESCRIPTION
**Pull request by forbesmyester**
RE: https://github.com/Quramy/tsuquyomi/issues/35

Here's the feature implemented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +126,9 @@ Alternatively, call the Ex command `:TsuquyomiReferences`.
 When a buffer is saved, Tsuquyomi checks syntax and semantics.
 And if it contains errors, Tsuquyomi show them to Vim quickfix window.
 
+### Disable Default Mappings
+If you do not want to use the default mappings please add `g:tsuquyomi_disable_default_mappings = 1` to your `.vimrc` file.
```
Comment (Quramy) [inline README.md]: `g:tsuquyomi_disable_default_mapings = 1` 
↓
`let g:tsuquyomi_disable_default_mappings = 1`

**Comment by Quramy** [comment]
I think it's a nice feature!

Please append `g:tsuquyomi_desable_default_mapping` section to `doc/tsuquyomi.txt` file's `VARIABLES` chapter.

**Comment by forbesmyester** [comment]
Thanks, how is it now?

**Comment by Quramy** [comment]
Ok, thanks :)

**Comment by forbesmyester** [comment]
:-)

**Pull request merged**